solutions/11.js

```diff
@@ -0,0 +1,7 @@
+const divisible = (num1,num2)=>{
```

Comment:
Only supposed to have one input number or parameter.

Reply:
i took second parameter because to make my programme dynamic, so that i can take any number, not only 7.